ERROR - Error while processing incoming UDP packet
java.lang.RuntimeException: Unexpected internal error FIXME!! Socket is closed
	at gov.nist.core.InternalErrorHandler.handleException(InternalErrorHandler.java:45)
	at gov.nist.javax.sip.stack.SIPClientTransaction.inviteClientTransaction(SIPClientTransaction.java:873)
	at gov.nist.javax.sip.stack.SIPClientTransaction.processResponse(SIPClientTransaction.java:546)
	at gov.nist.javax.sip.stack.SIPClientTransaction.processResponse(SIPClientTransaction.java:1431)
	at gov.nist.javax.sip.stack.UDPMessageChannel.processMessage(UDPMessageChannel.java:497)
	at gov.nist.javax.sip.stack.UDPMessageChannel.processIncomingDataPacket(UDPMessageChannel.java:414)
	at gov.nist.javax.sip.stack.UDPMessageChannel.run(UDPMessageChannel.java:258)
	at java.lang.Thread.run(Thread.java:637)
Caused by: java.net.SocketException: Socket is closed
	at java.net.DatagramSocket.send(DatagramSocket.java:578)
	at gov.nist.javax.sip.stack.UDPMessageChannel.sendMessage(UDPMessageChannel.java:677)
	at gov.nist.javax.sip.stack.MessageChannel.sendMessage(MessageChannel.java:237)
	at gov.nist.javax.sip.stack.SIPTransaction.sendMessage(SIPTransaction.java:754)
	at gov.nist.javax.sip.stack.SIPClientTransaction.sendMessage(SIPClientTransaction.java:488)
	at gov.nist.javax.sip.stack.SIPClientTransaction.inviteClientTransaction(SIPClientTransaction.java:871)
	... 6 more
ERROR - gov.nist.javax.sip.SipStackImpl.createListeningPoint(SipStackImpl.java:790) [Invalid argument address = 192.168.0.185 port = 5060 transport = udp]
ERROR - gov.nist.javax.sip.SipStackImpl.createListeningPoint(SipStackImpl.java:790) [Invalid argument address = 192.168.0.185 port = 5060 transport = udp]
ERROR - gov.nist.javax.sip.SipStackImpl.createListeningPoint(SipStackImpl.java:790) [Invalid argument address = 192.168.0.185 port = 5060 transport = udp]
ERROR - gov.nist.javax.sip.SipStackImpl.createListeningPoint(SipStackImpl.java:790) [Invalid argument address = 192.168.0.185 port = 5060 transport = udp]
ERROR - gov.nist.javax.sip.SipStackImpl.createListeningPoint(SipStackImpl.java:790) [Invalid argument address = 192.168.0.185 port = 5060 transport = udp]
ERROR - gov.nist.javax.sip.SipStackImpl.createListeningPoint(SipStackImpl.java:790) [Invalid argument address = 192.168.0.185 port = 5060 transport = udp]
ERROR - gov.nist.javax.sip.SipStackImpl.createListeningPoint(SipStackImpl.java:790) [Invalid argument address = 192.168.0.185 port = 5060 transport = udp]
ERROR - gov.nist.javax.sip.SipStackImpl.createListeningPoint(SipStackImpl.java:790) [Invalid argument address = 192.168.0.185 port = 5060 transport = udp]
ERROR - gov.nist.javax.sip.SipStackImpl.createListeningPoint(SipStackImpl.java:790) [Invalid argument address = 192.168.0.185 port = 5060 transport = udp]
ERROR - gov.nist.javax.sip.SipStackImpl.createListeningPoint(SipStackImpl.java:790) [Invalid argument address = 192.168.0.185 port = 5060 transport = udp]
ERROR - gov.nist.javax.sip.SipStackImpl.createListeningPoint(SipStackImpl.java:790) [Invalid argument address = 192.168.0.185 port = 5060 transport = udp]
ERROR - gov.nist.javax.sip.SipStackImpl.createListeningPoint(SipStackImpl.java:790) [Invalid argument address = 192.168.0.185 port = 5060 transport = udp]
ERROR - Error while processing incoming UDP packet
java.lang.RuntimeException: Unexpected internal error FIXME!! Socket is closed
	at gov.nist.core.InternalErrorHandler.handleException(InternalErrorHandler.java:45)
	at gov.nist.javax.sip.stack.SIPClientTransaction.inviteClientTransaction(SIPClientTransaction.java:873)
	at gov.nist.javax.sip.stack.SIPClientTransaction.processResponse(SIPClientTransaction.java:546)
	at gov.nist.javax.sip.stack.SIPClientTransaction.processResponse(SIPClientTransaction.java:1431)
	at gov.nist.javax.sip.stack.UDPMessageChannel.processMessage(UDPMessageChannel.java:497)
	at gov.nist.javax.sip.stack.UDPMessageChannel.processIncomingDataPacket(UDPMessageChannel.java:414)
	at gov.nist.javax.sip.stack.UDPMessageChannel.run(UDPMessageChannel.java:258)
	at java.lang.Thread.run(Thread.java:637)
Caused by: java.net.SocketException: Socket is closed
	at java.net.DatagramSocket.send(DatagramSocket.java:578)
	at gov.nist.javax.sip.stack.UDPMessageChannel.sendMessage(UDPMessageChannel.java:677)
	at gov.nist.javax.sip.stack.MessageChannel.sendMessage(MessageChannel.java:237)
	at gov.nist.javax.sip.stack.SIPTransaction.sendMessage(SIPTransaction.java:754)
	at gov.nist.javax.sip.stack.SIPClientTransaction.sendMessage(SIPClientTransaction.java:488)
	at gov.nist.javax.sip.stack.SIPClientTransaction.inviteClientTransaction(SIPClientTransaction.java:871)
	... 6 more
ERROR - Error while processing incoming UDP packet
java.lang.RuntimeException: Unexpected internal error FIXME!! Socket is closed
	at gov.nist.core.InternalErrorHandler.handleException(InternalErrorHandler.java:45)
	at gov.nist.javax.sip.stack.SIPClientTransaction.inviteClientTransaction(SIPClientTransaction.java:873)
	at gov.nist.javax.sip.stack.SIPClientTransaction.processResponse(SIPClientTransaction.java:546)
	at gov.nist.javax.sip.stack.SIPClientTransaction.processResponse(SIPClientTransaction.java:1431)
	at gov.nist.javax.sip.stack.UDPMessageChannel.processMessage(UDPMessageChannel.java:497)
	at gov.nist.javax.sip.stack.UDPMessageChannel.processIncomingDataPacket(UDPMessageChannel.java:414)
	at gov.nist.javax.sip.stack.UDPMessageChannel.run(UDPMessageChannel.java:258)
	at java.lang.Thread.run(Thread.java:637)
Caused by: java.net.SocketException: Socket is closed
	at java.net.DatagramSocket.send(DatagramSocket.java:578)
	at gov.nist.javax.sip.stack.UDPMessageChannel.sendMessage(UDPMessageChannel.java:677)
	at gov.nist.javax.sip.stack.MessageChannel.sendMessage(MessageChannel.java:237)
	at gov.nist.javax.sip.stack.SIPTransaction.sendMessage(SIPTransaction.java:754)
	at gov.nist.javax.sip.stack.SIPClientTransaction.sendMessage(SIPClientTransaction.java:488)
	at gov.nist.javax.sip.stack.SIPClientTransaction.inviteClientTransaction(SIPClientTransaction.java:871)
	... 6 more
